projects/OpenWebCalendar/default.nix

```diff
@@ -0,0 +1,16 @@
+{
+  pkgs,
+  lib,
+  sources,
+  ...
+}@args:
+{
+  metadata.subgrants = [ "OpenWebCalendar" ];
+  nixos.modules.services.open-web-calendar = {
+    name = "open-web-calendar";
+    module = "${sources.inputs.nixpkgs}/nixos/modules/services/web-apps/open-web-calendar.nix";
+    examples = { };
+    links = { };
```

Comment:
Since we're handling it already we may add a some links as we go.

Reply:
done, I took all the links from the authors reply to our survey